activities/ActivityTemplate/VueJS/locale.ini

```diff
@@ -0,0 +1,207 @@
+[*]
```

Comment:
Now that the `locale.ini` file is no longer empty, the [tutorial step 5](https://github.com/llaske/sugarizer/blob/dev/docs/tutorial/VueJS/step5.md#identify-strings-to-localize) should be updated.

Reply:
Okay I'll update the step.